Solution Week-1/Program_4.py

```diff
@@ -0,0 +1,12 @@
+#Write a python script to enter any number, if it is integer number, then check its armstrong or not. Print appropriate message if it is not armstrong.
+def arm():
+    b=input("Enter any  number >  ")
+    sum=0
+    for i in range(len(b)):

```

Comment:
here if we enter any alphabet then it returns an error.please solve this error.you can you .isdigit() function.

Reply:
In this code I have further type casted the input to integer value using int() ...